### PR TITLE
make a couple methods static

### PR DESCRIPTION
**Pull request by jmalkin**

### CODE REVIEW

File: src/test/java/org/apache/datasketches/filters/bloomfilter/DirectBitArrayRTest.java [hunk 1]
```diff
@@ -32,7 +32,7 @@
 
 public class DirectBitArrayRTest {
 
-  private Memory bitArrayToMemory(HeapBitArray ba) {
+  static private Memory bitArrayToMemory(HeapBitArray ba) {
```
Comment (leerho) [inline src/test/java/org/apache/datasketches/filters/bloomfilter/DirectBitArrayRTest.java:35]: Recommended practice is "private static"

File: src/test/java/org/apache/datasketches/filters/bloomfilter/DirectBitArrayTest.java [hunk 2]
```diff
@@ -30,7 +30,7 @@
 
 public class DirectBitArrayTest {
 
-  private WritableMemory bitArrayToWritableMemory(HeapBitArray ba) {
+  static private WritableMemory bitArrayToWritableMemory(HeapBitArray ba) {
```
Comment (leerho) [inline src/test/java/org/apache/datasketches/filters/bloomfilter/DirectBitArrayTest.java:33]: Recommended practice is "private static"

File: src/test/java/org/apache/datasketches/filters/bloomfilter/DirectBitArrayRTest.java (hunk 1 above)
Comment (leerho) [reply src/test/java/org/apache/datasketches/filters/bloomfilter/DirectBitArrayRTest.java:35]: See [Java language spec 8.4.3](https://docs.oracle.com/javase/specs/jls/se7/html/jls-8.html#jls-8.4.3)

**Review by leerho — CHANGES_REQUESTED** [review]

**Review by leerho — APPROVED** [review]

File: src/test/java/org/apache/datasketches/filters/bloomfilter/DirectBitArrayRTest.java (hunk 1 above)
Comment (jmalkin) [reply src/test/java/org/apache/datasketches/filters/bloomfilter/DirectBitArrayRTest.java]: fixed

**Pull request merged**